.docker/Dockerfile

```diff
@@ -26,4 +25,6 @@ RUN apt install -y \
     mailutils
 
 RUN php -r "copy('https://getcomposer.org/installer', '/tmp/composer-setup.php');" && php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer && rm -rf /tmp/composer-setup.php
+
+COPY .docker/docker_run_git.sh /tmp/
```

Comment:
moving this copy here to get better caching for the docker build

Reply:
FYI @jolelievre